src/Instances/Webhook.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+namespace RenokiCo\PhpK8s\Instances;
+
+class Webhook extends Instance
+{
+    /**
+     * Get the webhook name.
+     *
+     * @return string
+     */
+    public function getName(): string
```

Comment:
Classes that have `getXXXX` or `setXXXX` should be added only if their key is not in the same row. For example, here, `getName` and perhaps other functions are already covered by [the `__call` implementation](https://github.com/renoki-co/php-k8s/blob/3.1.8/src/Traits/Resource/HasAttributes.php#L183-L208), so I guess they are redundant.

`addRule` and `addRules` are good here.

Reply:
Thank you for letting me know, I didn't notice the implementation for the `__call` method 🙂